sky/spot/spot_utils.py

```diff
@@ -655,7 +655,7 @@ def format_job_table(
         f'{count} {status}' for status, count in sorted(status_counts.items())
     ])
     if status_str:
-        status_str = f'In progress jobs: {status_str}'
+        status_str = f'In progress tasks: {status_str}'
```

Comment:
Should this be tasks or jobs? Seems different from L660?

Reply:
(not for this PR but for future) Was thinking we could alternatively show PENDING/RUNNING "jobs"? Since each pipeline job has at most 1 in-progress task?

Reply:
This is a good point! We can consider doing that. Let's wait for user feedback for this. ; )